helm-charts/fluentd-es/templates/fluentd-es.yaml

```diff
@@ -56,6 +56,31 @@ roleRef:
   name: {{  template "fluentd-es.fullname" . }}
   apiGroup: ""
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fluentd-es-metrics
+  namespace: logging
+  labels:
+    k8s-app: fluentd-es
+    version: {{ .Values.image.tag }}
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+    app: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  ports:
+  - name: metrics
+    port: 24231
+    protocol: TCP
+    targetPort: 24231
+  selector:
```

Comment:
This works but on a closer read I think I mistakenly picked these two labels. I think the selector here should be the same as the `DaemonSet` selector on L98-101.

I'm not sure which one is better, however, it certainly shouldn't select based on hardcoded values when `app` and `release` are both generated (also, `app` and `release` specifically are equal, see the `DaemonSet` spec below).

Perhaps the following two would work nicely?
```
     app: {{ .Release.Name }}
     version: {{ .Values.image.tag }}
```
This combination seems unique enough.